examples/vision/transformer_in_transformer.py

```diff
@@ -0,0 +1,328 @@
+"""
+Title: Image classification with TNT(Transformer in Transformer)
+Author: [ZhiYong Chang](https://github.com/czy00000)
+Date created: 2021/10/25
+Last modified: 2021/11/06
+Description: Implementing the Transformer in Transformer (TNT) model for image classification.
+"""
+
+"""
+## Introduction
+This example implements the [TNT](https://arxiv.org/abs/2103.00112)
```

Comment:
This looks like it is not precisely the architecture hyper-parameters from the paper correct? (depth of 5 blocks vs 12?)

If that's right, might be helpful to introduce that somewhere in this introduction. "To keep training time reasonable will train and test a smaller model than is in the paper (XX params vs XX params)." Something like that.